### DISCUSSION TITLE
realtime.schema_migrations does not exist

### DISCUSSION THREAD

**Discussion by ricardosikic** [supabase/supabase#13097] [Questions]
Hi, I am running the `SUPABASE_SCANNER_BUFFER_SIZE=200MB supabase start` command after dumping `--column-inserts` the problem is that it returns `Error: ERROR: relation "realtime.schema_migrations" does not exist (SQLSTATE 42P01)`. 

```
2023/03/20 01:17:35 PG Recv: {"Type":"ReadyForQuery","TxStatus":"I"}
2023/03/20 01:17:35 PG Send: {"Type":"Parse","Name":"","Query":"INSERT INTO realtime.schema_migrations (version, inserted_at) VALUES (20230228184745, '2023-03-04 18:48:30')","ParameterOIDs":null}
2023/03/20 01:17:35 PG Send: {"Type":"Describe","ObjectType":"S","Name":""}
2023/03/20 01:17:35 PG Send: {"Type":"Sync"}
2023/03/20 01:17:35 PG Recv: {"Type":"ErrorResponse","Severity":"ERROR","SeverityUnlocalized":"ERROR","Code":"42P01","Message":"relation \"realtime.schema_migrations\" does not exist","Detail":"","Hint":"","Position":13,"InternalPosition":0,"InternalQuery":"","Where":"","SchemaName":"","TableName":"","ColumnName":"","DataTypeName":"","ConstraintName":"","File":"parse_relation.c","Line":1371,"Routine":"parserOpenTable","UnknownFields":null}
2023/03/20 01:17:35 PG Recv: {"Type":"ReadyForQuery","TxStatus":"I"}
2023/03/20 01:17:35 PG Send: {"Type":"Terminate"}
Error: ERROR: relation "realtime.schema_migrations" does not exist (SQLSTATE 42P01)
```

**Comment by sweatybridge** [comment]
Hello, could you comment out any inserts into realtime.schema_migrations table? I believe those are not necessary for local dev or migrating to another hosted project. 

Also which version of CLI are you using? I remember releasing a similar fix recently.

**Reply by ricardosikic** [reply]
Hi, @sweatybridge i did that when the error came out but for some reason did not fixed the problem, my version i believe is: `1.42.7
`

**Reply by sweatybridge** [reply]
Could you be more specific with the error message after commenting out those lines?

**Comment by customkandura** [comment]
the migrations of realtime init sql to create schema might doesn't exist, also try set search_path of your role to that schema
